app/src/main/java/com/asfoundation/wallet/subscriptions/SubscriptionApiMockedImpl.kt

```diff
@@ -11,8 +11,12 @@ class SubscriptionApiMockedImpl : SubscriptionApiMocked {
       val subs1 = Subscription("Real Boxing", "com.vividgames.realboxing",
           "http://pool.img.aptoide.com/bds-store/59a7b62a169a832e96dbd7df82d6e3cc_icon.png",
           BigDecimal.TEN, "€", "EUR", true, "PayPal",
-          PAYPAL_ICON_URL, getFutureDate(), getPastDate(), getPastDate(), "Month")
-      it.onSuccess(listOf(subs1))
+          PAYPAL_ICON_URL, getFutureDate(), getPastDate(), getPastDate(), null, "Month")
```

Comment:
Should we have the interface with the "Mocked" indication since the mocked part is the implementation?

Reply:
[MOB-46] Updated screens after review feedback